Assets/HoloToolkit/Boundary/Scripts/SceneContentAdjuster.cs

```diff
@@ -28,8 +40,7 @@ private void Awake()
             }
 
 #if UNITY_2017_2_OR_NEWER
-            // A Stationary TrackingSpaceType doesn't need any changes for an object.
-            if (XRDevice.GetTrackingSpaceType() == TrackingSpaceType.Stationary || !XRDevice.isPresent)
+            if (!XRDevice.isPresent)
```

Comment:
Should we be checking is the headset is opaque?

Reply:
Hmm...I don't think so. Although the floor is only set in opaque headsets right now, that doesn't necessarily have to be the case forever. The height offset really only comes from Stationary vs RoomScale.

Reply:
Could we add a note about that here please?